src/foolscap/test/test_banana.py

```diff
@@ -1878,7 +2044,7 @@ def test_incoming1(self):
         b = TokenBanana()
         b.connectionMade()
         vdict = {1: 'list', 2: 'tuple', 3: 'dict'}
-        keys = vdict.keys()
+        keys = list(vdict.keys())
         keys.sort()
         setVdict = [tOPEN(0),'set-vocab']
         for k in keys:
```

Comment:
`for k in sorted(vdict.keys()):` perhaps

Reply:
Yes. Makes sense. Changed.